src/components/Breadcrumb/Breadcrumb.vue

```diff
@@ -101,13 +105,24 @@ export default {
 			type: Boolean,
 			default: false,
 		},
+		/**
+		 * Specify the open state of the actions dropdown menu
+		 */
+		open: {
+			type: Boolean,
+			default: false,
+		},
 	},
 	data() {
 		return {
 			/**
 			 * Variable to track if we hover over the breadcrumb
 			 */
 			hovering: false,
+			/**
+			 * Variable to the open state of the Actions menu
+			 */
+			opened: this.open,
```

Comment:
This won't work if the open var changes on the parent.
You just init it on component creation, but you also need to watch.

I think just emitting again the `open:update` event should be enough :) so that you can directly do `<Breadcrumb :open.sync="open" />`

So I would just do
```
<Actions :force-menu="forceMenu" :open="open" @update:open="onOpenChange">
...

methods: {
	onOpenChange(open) {
		this.$emit('update:open, open)
	}
}
```

Reply:
With this change it doesn't open anymore when I do ` this.$refs.actions.opened = true` in the Breadcrumbs component. How would you open it then?

Reply:
It doesn't open at all?
Weird! `opened` is a autonomous variable inside Actions. So changing it should work out of the box :thinking: Can you debug if it toggle twice maybe?

Reply:
Well, `this.$refs.actions.opened = true` is a bit misleading. It rather is `this.$refs.actions` is the breadcrumb component, not the Actions themself. In the `Breadcrumbs` don't have a refs to the Actions directly, since they are rendered by the `Breadcrumb` component.

Reply:
Just tested, and it worked :man_shrugging:

Reply:
![Peek 30-03-2020 10-37](https://user-images.githubusercontent.com/14975046/77892185-79970500-7272-11ea-9a70-8309a7712e3d.gif)

Reply:
Though, we should force a max-height :see_no_evil: 
Shall we do that for the whole Actions component or here?

Reply:
Like this 201b3c305204a4b85b0c8d70de6773db0a83afdb then?

Reply:
> Though, we should force a max-height 🙈
> Shall we do that for the whole Actions component or here?

I had a max-height in the initial PR, but I think we agreed to put it in the Actions menu.